src/main/kotlin/no/nav/pensjon/kalkulator/simulering/client/simulator/map/SimulatorSimuleringType.kt

```diff
@@ -10,6 +10,7 @@ import no.nav.pensjon.kalkulator.simulering.SimuleringType
 enum class SimulatorSimuleringType(val externalValue: String, val internalValue: SimuleringType) {
 
     ALDERSPENSJON("ALDER", SimuleringType.ALDERSPENSJON),
+    AFP_ETTERF_ALDER("AFP_ETTERF_ALDER", SimuleringType.AFP_ETTERF_ALDER),
```

Comment:
Foreslår at enum-verdi-navnet endres til `PRE2025_OFFENTLIG_AFP_ETTERFULGT_AV_ALDERSPENSJON`.
Dette fordi vi har brukt fullstendige ord for de andre typene, og det beskriver hvilken AFP-type dette gjelder.